src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -8,27 +8,40 @@
 
 import java.util.Set;
 
+import com.revrobotics.CANEncoder;
 import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
 
 import edu.wpi.first.wpilibj.PWM;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.Subsystem;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
+import frc.robot.Constants;
+
 public class Shooter extends SubsystemBase {
     CANSparkMax motor1, motor2, motor3;
+    CANEncoder shooterEncoder;
+    CANSparkMax FLUP;
```

Comment:
Flup is not a constant so it should not be all capitals. Please change "FLUP" to "Flup"